<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-IVCEMNE" failures="1" tests="1" name="Listener Practice" time="0.106" errors="0" timestamp="2025-08-22T21:52:43 IST">
  <testcase classname="TestNG_Basics.DataProvider_Practice" name="getDetails" time="0.0">
    <failure type="java.lang.RuntimeException" message="java.lang.AssertionError: expected [true] but found [false]">
      <![CDATA[java.lang.RuntimeException: java.lang.AssertionError: expected [true] but found [false]
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
Caused by: java.lang.AssertionError: expected [true] but found [false]
at TestNG_Basics.DataProvider_Practice.getData(DataProvider_Practice.java:33)
... 25 more
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getDetails -->
</testsuite> <!-- Listener Practice -->
